lib/tasks/brexit_checker_unpublisher.rake

```diff
@@ -0,0 +1,42 @@
+namespace :brexit_checker do
+  desc "Unpublish the brexit checker and redirect to new slugs" do
+    task unpublish: [:environment] do
+      BrexitCheckerUnpublisher.call
+    end
+  end
+end
+
+class BrexitCheckerUnpublisher
+  def self.list
+    [{
+       content_id: "1102fd2b-7b29-43f5-889b-3e781e09971f",
+       new_base_path: "/transition-check/questions",
+       old_base_path: "/get-ready-brexit-check/questions",
+     },
+     {
+       content_id: "2c73a7e4-2473-4215-8257-04ebe73ca1bc",
+       new_base_path: "/transition-check/results",
+       old_base_path: "/get-ready-brexit-check/results",
+     },
+     {
+       content_id: "0593033b-f713-41c5-bc67-90545091805c",
+       new_base_path: "/transition-check/email-signup",
+       old_base_path: "/get-ready-brexit-check/email-signup",
+     }]
+  end
+
+  def self.call
+    list.each do |to_unpublish|
+      Services.publishing_api.unpublish(to_unpublish[:content_id],
+                                        type: "redirect",
+                                        explanation: "renamed to transition-check",
```

Comment:
Will an exploitation be accepted for this type of unpublished? The docs seem to reckon it's only relevant for, withdrawal or gone? plus is only if it gets shown on a page?
```
 explanation (conditionally required)

    Required for a type of "withdrawal", Optional for a type of "gone".
    Message that will be displayed publicly on the page that has been unpublished.
```

Reply:
https://docs.publishing.service.gov.uk/apis/publishing-api/api.html#post-v2contentcontent_idunpublish

Reply:
I'll remove it